include/appfwk/CommandFacility.hpp

```diff
@@ -38,69 +38,42 @@
   }
 
 namespace dunedaq::appfwk {
-class DAQProcess; // forward declaration
+class DAQModuleManager; // forward declaration
 
 /**
- * @brief Interface needed by Application Framework for CCM command handling
+ * @brief Interface needed by Application Framework for command handling
  */
 class CommandFacility
 {
 public:
-  /**
-   * @brief Singleton pattern; get a handle to the CommandFacility
-   * @return Reference to the CommandFacility
-   */
-  static CommandFacility& handle()
-  {
-    if (!handle_)
-      handle_.reset(new CommandFacility());
-    return *handle_;
-  }
+  CommandFacility(std::string /*uri*/) {}
```

Comment:
It may be a TODO item, but I think it would be good to maintain Doxygen comments in the code